seawatch_registration/Tests/test_views.py

```diff
@@ -21,6 +21,7 @@ def setUp(self) -> None:
         self.url_edit_profile = reverse('edit_profile')
         self.url_show_profile = reverse('show_profile')
         self.url_add_document = reverse('add_document')
+        self.url_add_skills = reverse('add_skills')
 
     def test_views__add_profile__get__should_redirect_to_login_when_user_is_not_logged_in(self):
```

Comment:
the tests, which are implemented to test the skill selection, should also be implemented for the profile view.

Reply:
fixed with https://github.com/innoq/seawatch_planner/pull/24/commits/6d89a441120b2d1e141a07f46b988a476091a0a0